web/client/components/contextcreator/ConfigureThemes.jsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+
+import React from 'react';
+import Select from 'react-select';
+import {ControlLabel} from 'react-bootstrap';
+import Message from '../I18N/Message';
+
+
+const ConfigureThemes = ({
+    themes,
+    setSelectedTheme = () => {},
+    selectedTheme = "default"
+}) => {
+    return (
+        <div className="configure-themes-step">
+            <div className="choose-theme">
+                <ControlLabel><Message msgId="contextCreator.configureThemes.themes"/></ControlLabel>
+                <Select
+                    onChange={({value})  => setSelectedTheme(value)}
+                    value={selectedTheme}
+                    options={themes}
+                    disabled={false}
+                    noResultsText="contextCreator.configureThemes.noThemes"
+                />
```

Comment:
I did not test this but should the change needed for the previous comment
```suggestion
                <Select
                    onChange={(option)  => setSelectedTheme(option?.theme)}
                    value={selectedTheme}
                    options={themes.map(theme => ({ value: theme.id, label: theme.label || theme.id, theme }))}
                    disabled={false}
                    noResultsText="contextCreator.configureThemes.noThemes"
                />
```

Reply:
i think we need to use option.id